vips/color.c

```diff
@@ -7,3 +9,31 @@ int is_colorspace_supported(VipsImage *in) {
 int to_colorspace(VipsImage *in, VipsImage **out, VipsInterpretation space) {
 	return vips_colourspace(in, out, space, NULL);
 }
+
+// https://libvips.github.io/libvips/API/8.6/libvips-colour.html#vips-icc-transform
+int optimize_icc_profile(VipsImage *in, VipsImage **out, int isCmyk) {
```

Comment:
didn't you want to remove the _isCmyk_ parameter?

Reply:
eventually, didn't want to mess to much with the C code at once